Sources/Atomics/CMakeLists.txt

```diff
@@ -22,8 +22,12 @@ add_library(Atomics
   DoubleWord.swift)
 set_target_properties(Atomics PROPERTIES
   INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_Swift_MODULE_DIRECTORY})
-target_compile_definitions(Atomics PRIVATE
+target_compile_definitions(Atomics PUBLIC
```

Comment:
Is `ENABLE_DOUBLEWIDE_ATOMICS` meant to be defined by anyone using `Atomics`?

Reply:
I *guess* so -- otherwise clients would have a difficult time determining whether double-wide atomics were available.

Ideally we would be able to just force-enable double-wide atomics though.